content/posts/newsletter/2019-10-01.md

```diff
@@ -0,0 +1,123 @@
+---
+title: The XMPP Newsletter, 01 Oct 2019: FOSDEM 2020, modernization of XMPP, peer networks
+save_as: 2019/10/newsletter-01-october/index.html
+url: 2019/10/newsletter-01-october/
+date: 2019-10-01 09:00
+author: nyco
+blog_id: newsletter
+category: Newsletter
+---
+
+Welcome to the XMPP newsletter covering the month of September.
+
+New this month: we've made explicit that this newsletter can be shared and adapted as defined by the [CC by-sa 4.0 license](https://creativecommons.org/licenses/by-sa/4.0/), and we've added the credits as this is a community effort.
+
+Be kind, inform your friends and colleagues: forward this newsletter!
+
+Please [submit your XMPP/Jabber articles, tutorials or blog posts](https://wiki.xmpp.org/web/News_and_Articles_for_the_next_XMPP_Newsletter) on our wiki.
+
+<form style="padding: 10px; text-align:center; margin-bottom: 30px;"
+      action="https://tinyletter.com/xmpp" method="post" target="popupwindow"
+      onsubmit="window.open('https://tinyletter.com/xmpp', 'popupwindow',
+      'scrollbars=yes,width=800,height=600');return true">
+<p><label for="tlemail">Subscribe to receive the next edition in your inbox</label></p>
+<p><input type="text" placeholder="Email address" name="email" id="tlemail" /></p>
+<input type="hidden" value="1" name="embed"/>
+<input type="submit" style="padding: 10px; border-radius: 5%" value="Subscribe" />
+</form>
+
+## Articles
+
+Ralph Meijer, chairman of the Board of Directors of the XSF, has written an introductory piece about "[XMPP Message Attaching, Fastening, References](https://ralphm.net/blog/2019/09/09/fastening)", specifications currently in progress.
+
+In case of internet shut down, whether by disaster or voluntarily, peer networks are useful. [Monal uses Airdrop](http://monal.im/blog/xmppsignalbluetoothp2p-wifi-serverless-chat/) (wifi and bluetooth) along with XMPP and OMEMO end-to-end encryption.
+
+![Monal and Airdrop screenshots](https://framapic.org/Qg1mv43w1673/akPmtepp0nfH.png)
+
+JabberFr, the French XMPP/Jabber community, has again [translated the XMPP newsletter in French](https://news.jabberfr.org/2019/09/lettre-dactualite-xmpp-du-3-septembre-2019/), merci beaucoup.
+
+We have started a very minimalistic [communication guide](https://wiki.xmpp.org/web/Basic_communication_guide_for_XMPP_techies) to help promoting a project through social networks and other means such as blog posts. Could be worth for those involved in XMPP projects and would like to get ideas on how to reach out to different communities.
+
```

Comment:
"could be worth" -> "could be valuable"?

Reply:
Of course! Fixed.